e2e/docs/E2E_FRAMEWORK.md

```diff
@@ -0,0 +1,229 @@
+# Pocket 1.0 E2E Testing Framework
```

Comment:
@dylanlott It seems like a lot of changes made to this file were reverted.

1. Some of the headers are bolded again
2. Ommit toc is reverted
3. Dates were introduced back
4. All the nits and suggestions were added back

Please re-review when you have a chance

Reply:
Addressed in https://github.com/pokt-network/pocket/pull/663